Seminars/Sem.02/Pract.02/Task07.cpp

```diff
@@ -25,7 +25,7 @@ struct System
 
 void inputUsers(System &newSystem)
```

Comment:
I do not understand the logic here. As I can see, you want to save the whole system to file, therefore you do not change it. Why do you change the countOfStudents . Re-think the logic here. Why the for cycle is looping to the maximum possible number of users. It should be to the current number of users in the system